lib/beacon/application.ex

```diff
@@ -14,6 +14,8 @@ defmodule Beacon.Application do
       Beacon.Repo
     ]
 
+    :ets.new(:beacon_pages, [:set, :named_table, :public, read_concurrency: true, write_concurrency: true])
```

Comment:
Unless you plan to write to this table a lot, I would remove `write_concurrency` from this list. My assumption is that this is a table vastly dominated by reads. Without the option, you can of course still write to the table concurrently, it is just more expensive to do so.

Reply:
It won't be written very often. I'm gonna remove that opt.